master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -81,15 +82,15 @@ func (e *ExperimentConfigV0) Scan(src interface{}) error {
 	return nil
 }
 
-// Description is a container struct for handling runtime defaults. It has to be a container so that
+// Name is a container struct for handling runtime defaults. It has to be a container so that
 // it can be responsible for allocating the nil pointer if one is not provided.  It would be nice if
-// you could use `type Description *string` but go won't let you create methods on such a type.
-type Description struct {
+// you could use `type Name *string` but go won't let you create methods on such a type.
+type Name struct {
```

Comment:
maybe ask @rb-determined-ai how he feels about this/run it by him since he's landing the last bit of expconf/fixing it up now

Reply:
we have chatted about it. let me know if you have any other comments @rb-determined-ai

Reply:
ok I just forget everything, sgtm.